### DISCUSSION TITLE
SNES-like sound bank for Timidity

### DISCUSSION THREAD

**Discussion by Alucard648** [WohlSoft/PGE-Project#433] [General]
I have seen mentioning about this side project during earlier stages of PGE development, but now it went all quiet. Any thoughts?

**Comment by Wohlstand** [comment]
That bank is still be available, and newer version you can find here: https://yadi.sk/d/bky-syShwpTRH
Sometimes I testing it and updating any instruments in it, but rare

**Comment by Alucard648** [comment]
Any instructions on how to set it up to work with SDL Mixer and PGE Musplay?
EDIT: Just thrown the folder into PGE directory and it worked. But all in all still leaving this ticket open as the sound bank still seems incomplete and has some issues like missing instruments.
Noted during testing:
1. Church organ patch sounds like a music box.
2. Harp patch sounds like pizzcato strings.
3. Grand piano patch sounds like Honkey-Tonk piano from SMW.
